confignode/src/main/java/org/apache/iotdb/confignode/conf/ConfigNodeConfig.java

```diff
@@ -1113,4 +1116,8 @@ public long getDataRegionRatisLogMax() {
   public void setDataRegionRatisLogMax(long dataRegionRatisLogMax) {
     this.dataRegionRatisLogMax = dataRegionRatisLogMax;
   }
+
+  public boolean isPartitionDebugEnabled() {
```

Comment:
How to update this parameter?

Reply:
This parameter is only maintained by us. Since printing PartitionTable is time-comsuming, we only set this parameter to true if there occur serious bugs.